R/datatables.R

```diff
@@ -932,9 +932,17 @@ DTDependencies = function(style) {
 
 # translate DataTables classes to Bootstrap table classes
 DT2BSClass = function(class) {
-  class = unlist(strsplit(class, '\\s+'))
-  if ('display' %in% class)
+  if (is.list(class)) {
+    chosen_classes = Filter(function(x) isTRUE(x), class)
+    class = names(chosen_classes)
```

Comment:
Perhaps this suffices?

```suggestion
    class = names(which(unlist(class)))
```